pilot/pkg/model/authentication.go

```diff
@@ -97,13 +110,75 @@ func (policy *AuthenticationPolicies) addRequestAuthentication(configs []Config)
 	}
 }
 
+func apiModeToMutualTLSMode(mode v1beta1.PeerAuthentication_MutualTLS_Mode) MutualTLSMode {
+	switch mode {
+	case v1beta1.PeerAuthentication_MutualTLS_DISABLE:
+		return MTLSDisable
+	case v1beta1.PeerAuthentication_MutualTLS_PERMISSIVE:
+		return MTLSPermissive
+	case v1beta1.PeerAuthentication_MutualTLS_STRICT:
+		return MTLSStrict
+	default:
+		return MTLSUnknown
+	}
+}
+
 func (policy *AuthenticationPolicies) addPeerAuthentication(configs []Config) {
+	foundNamespaceMTLS := make(map[string]v1beta1.PeerAuthentication_MutualTLS_Mode)
+
 	for _, config := range configs {
 		policy.peerAuthentications[config.Namespace] =
 			append(policy.peerAuthentications[config.Namespace], config)
+
+		// Mesh & namespace level policy must be named as `default` (config_consts.DefaultAuthenticationPolicyName)
```

Comment:
Can we revert this change? This will break multiple control plane.. why do we need an arbitrary restriction?

Reply:
We want to enforce singleton. Why does it break multiple control plane btw?

Reply:
i think it's already there, this limitation.

with multi control plane, will they be lived in the same ns? if they are different, we can have two `rootNS`, and each is named as default. will that work?

Reply:
Because its no longer a singleton with multiple control planes. I should be able to  define one policy for ns `foo` with `istio.io/rev=istiod-1` and another in ns `foo` with `istio.io/rev=istiod=2`. Then it is still a "singleton" from the perspective of each namespace.

I suppose it does *break* this, it just means you cannot do canarys of this config

Reply:
Hmm, ok I change this to check selector. Need to send out different PR to drop validation if that the case, though I don't see how can we enforce the singleton then